Automattic/vip-go-mu-plugins
# Bump vip-dashboard submodule to a9f3d8e

## Description

This PR updates the links to wpvip.com and docs.wpvip.com in the VIP page for WordPress Dashboard.

## Changelog Description

### VIP Dashboard updated

We have updated VIP Dashboard Page to include the current links to wpvip.com and docs.wpvip.com

https://github.com/Automattic/vip-go-mu-plugins/pull/1931

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Go to `wp-admin` > `VIP`
1. Hover over some links.
1. Verify the links to `vip.wordpress.com` now point to `wpvip.com` and the `vip.wordpress.com/documentation` to `docs.wpvip.com`

props @gamebits
